docs/explanation/lex-persona.md

```diff
@@ -0,0 +1,628 @@
+            "imageX": 390.0,
+            "imageY": 710.0,
+            "imageWidth": 150.0,
+            "imageHeight": 80.0
+        },
+        {
+            "imagePage": -1,
+            "imageX": 180.0,
+            "imageY": 355.0,
+            "imageWidth": 150.0,
+            "imageHeight": 80.0
+        }
+    ]
+```
+
+#### 4- Launch the workflow process
+
+**Endpoint** : api/workflows/{{workflow_id_created}}/
+
+**Method** : PATCH
+
+**Headers**:
+- Authorization : Bearer {{token}}
+- Content-Type : application/json
+
+**Body Type** :  JSON
+
+Payload :
+```json
+{
+    "workflowStatus": "started"
+}
+```
+Response :
+```json
+{
+    "created": 1688993660520,
+    "currentRecipientEmails": [
+        "<signer_email_address>@mail.com"
+    ],
+    "currentRecipientUsers": [],
+    "description": "91 Allée, Voie de Presbourg, 60282 Vénissieux",
+    "email": "<user_email>@mail.fr",
+    "firstName": "Sam",
+    "groupId": "grp_9woWtcSt5KiAzyq********",
+    "id": "wfl_4yzLat7tDUQ8JwF********",
+    "jobOperation": "processWorkflow",
+    "lastName": "P******",
+    "logs": [],
+    "name": "Aerodynamic Silk Bottle",
+    "notifiedEvents": [
+        "recipientRefused",
+        "recipientFinished",
+        "workflowStopped",
+        "workflowFinished"
+    ],
+    "progress": 0,
+    "started": 1688994896088,
+    "steps": [
+        {
+            "allowComments": true,
+            "hideAttachments": false,
+            "hideWorkflowRecipients": true,
+            "id": "stp_D5KgBHk63MkhdGw********",
+            "invitePeriod": 86400000,
+            "isFinished": false,
+            "isStarted": true,
+            "logs": [
+                {
+                    "created": 1688994896088,
+                    "operation": "start"
+                }
+            ],
+            "maxInvites": 5,
+            "recipients": [
+                {
+                    "consentPageId": "cop_NzBEGbgTkj7v5tq********",
+                    "country": "FR",
+                    "email": "<signer_email_address>@mail.com",
+                    "firstName": "John",
+                    "lastName": "Doe",
+                    "phoneNumber": "+33 7 ** ** ** **",
+                    "preferredLocale": "fr"
+                }
+            ],
+            "requiredRecipients": 1,
+            "sendDownloadLink": true,
+            "stepType": "signature",
+            "validityPeriod": 8553600000
+        }
+    ],
+    "tenantId": "ten_9H4TvNNQFfrLBXH********",
+    "updated": 1688994896088,
+    "userId": "usr_P5YBCEeRqY8jKGG********",
+    "viewAuthorizedGroups": [
+        "grp_9woWtcSt5KiAzyq********"
+    ],
+    "viewAuthorizedUsers": [],
+    "watchers": [],
+    "workflowStatus": "started"
+}
+```
+We can observe that the "workflowStatus" value is set to "started" and the "progress" is set to 0.
+
+At this point, the recipients will receive through email a notification that a document requires to be signed by them.
```

Comment:
Is there a way to not send an email but just retrieve the link to sign the document and display it to the user on a UI ?

Reply:
Indeed, there is a way to create the invitation link which will allow us to display it to the user on UI.
The endpoint is : /api/workflow/{{id_workflow}}/invite.

The way is to set it up when we create the workflow, where we set to the `maxInvites` field the value 0.